src/components/PopupWithDeleteCard.js

```diff
@@ -0,0 +1,22 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDeleteCard extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.